uui-components/src/overlays/Dropdown.tsx

```diff
@@ -223,6 +223,14 @@ export class Dropdown extends React.Component<DropdownProps, DropdownState> {
         });
     }
 
+    private handleBodyKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+        console.log('keydown ' + e.key);
+        if (e.key === 'Escape' && this.isOpened()) {
+            e.stopPropagation();
```

Comment:
we can't use `e.stopPropagation()` in our library, because it will broke user listners, which also can be subscribed on this event.